src/zenml/zen_stores/migrations/versions/8a64fbfecda0_add_num_outputs_to_run_step.py

```diff
@@ -0,0 +1,45 @@
+"""Add num_outputs to run step [8a64fbfecda0].
+
+Revision ID: 8a64fbfecda0
+Revises: 5330ba58bf20
+Create Date: 2022-11-08 16:20:35.241562
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "8a64fbfecda0"
+down_revision = "5330ba58bf20"  # 0.21.1
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("pipeline_run", schema=None) as batch_op:
+        batch_op.alter_column(
+            "num_steps", existing_type=sa.INTEGER(), nullable=True
+        )
+
+    with op.batch_alter_table("step_run", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("num_outputs", sa.Integer(), nullable=True)
```

Comment:
are sa.Integer and sa.INTEGER the same thing?

Reply:
Great catch, they are not the same, `sa.INTEGER` will fail if the underlying DB doesn't have integer as data types. TL;DR: Always use CamelCase types, never UPPERCASE, see https://docs.sqlalchemy.org/en/14/core/type_basics.html#the-camelcase-datatypes